js/single_search.js

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Add [this](https://github.com/upeipls/UPLS/blob/4f6ba2191d46111c29551f71b8f9f2aeb47a19e4/js/edit_header.js#L1-L10), same as previous comment.